config/static/nodes/cgroup-v2/kustomization.yaml

```diff
@@ -2,5 +2,3 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```

Comment:
- Remove this file from the repository.
- Add this path to the .gitignore contents.
- Generate the file from the Makefile rule.